### PR TITLE
Bookstore: styling

### PR DESCRIPTION
**Pull request by Rocio01**
Greetings

I added styling, and I used Zeplin as a collaboration tool.

Kind regards

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -8,6 +8,8 @@
 - Add a book.
 - Remove a selected book.
 
+![screenshot](./Captura.PNG)
+
 
 ## Built With
```
Comment (SarvarKh) [inline README.md:14]: **[OPTIONAL]** You have well crafted your README.md. It really looks nice and clean!:champagne:

You can also consider those options to make more professional (take a look at [Microverse standard template](https://github.com/microverseinc/readme-template)):
- [ ] separate Major languages, Frameworks, Technologies.

For instance:
- Major languages: JavaScript, HTML, CSS
- Frameworks: React...

- [ ] add live demo link (you can easily upload it to netlify). If you need some tutorial, please check out this amazing 22 min video from Brad Traversy on [YouTube](https://www.youtube.com/watch?v=bjVUqvcCnxM&t=334s). Watching only the first half would be more than enough:wink:

File: src/App.css [hunk 2]
```diff
@@ -1,7 +1,242 @@
 body {
```
Comment (SarvarKh) [inline src/App.css:1]: Your style looks almost identical to the one shown in the Zeplin design.

However, one of the project requirements was Your final design should match Zeplin's design.
Therefore, we must stick to the requirement. Currently, your Bookstore app is missing pale grey background around your main app.

- [ ] Please add it.

Here is how your app looks like
Screen Shot 2021-11-06 at 02 17 45

Zeplin's design:
Screen Shot 2021-11-06 at 02 18 26

File: src/App.css [hunk 3]
```diff
@@ -1,7 +1,242 @@
...
+  font-style: normal;
+  line-height: normal;
+  letter-spacing: normal;
+  color: #0290ff;
+}
+
+.nav {
+  padding-left: 6% !important;
+  background-color: #fff;
+}
```
Comment (SarvarKh) [inline src/App.css:177]: Your final design should match Zeplin's design.

- [ ] please make sure your navbar looks exactly the same as in the Zeplin (currently it is missing avatar icon, some not capitalized, different colors, content and etc.);

Your navbar currently looks like
Screen Shot 2021-11-06 at 02 15 12
:

File: README.md [hunk 4]
```diff
@@ -8,6 +8,8 @@
 - Add a book.
 - Remove a selected book.
 
+![screenshot](./Captura.PNG)
+
```
Comment (SarvarKh) [inline README.md:12]: Your README looks great, but your MIT license link is not valid.

- [ ] please fix it;

File: src/App.css [hunk 5]
```diff
@@ -1,7 +1,242 @@
...
+  line-height: normal;
+  letter-spacing: -0.18px;
+  color: #888;
+}
+
+.form-container {
+  margin-left: 6.25rem;
+  width: 75rem;
+  padding: 0;
+}
```
Comment (SarvarKh) [inline src/App.css]: Your final design should match Zeplin's design.

- [ ] Please add horizontal line missing between the form and book cards container the same as in the Zeplin design;
- [ ] Please make sure your footer and form design matches the one given in Zeplin;
- [ ] no margin at the bottom of the form;
- [ ] different form content/placeholder;
- [ ] no box-shadow at right-bottom of the main app;
- [ ] total width of the form shall be stretched and be the same as the total width of the book card container;

**Review by SarvarKh — CHANGES_REQUESTED** [review]
Hi @Rocio01 ,

You have done a great job so far! 👏 
:heavy_check_mark: Your repository looks professional with descriptive README and commits;
:heavy_check_mark: Both Github and Git Flows were used correctly.

In order to go to the next project, you shall work on a few issues blocking your project from excellence.
You have well-crafted your site and are one step behind its completeness :trophy:

Implementing those given comments might be unpleasant, but your and my objectives are to make you a Master😎 and one of the Project requirements is your styling must be EXACTLY the same as the one provided on Zeplin.
So, do your best and give it the last shot🦾🙏

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by afanimartin — CHANGES_REQUESTED** [review]
Hi @Rocio01, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
Please add horizontal line missing between the form and book cards container the same as in the Zeplin design;
 - Please make sure your footer and form design matches the one given in Zeplin;
 - Remove the margin at the bottom of the form;
 - Please remove the box-shadow at right-bottom of the main app;
 - The total width of the form should not be stretched and be the same as the total width of the book card container. Please fix that.

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by Rocio01** [comment]
Greetings, 

I did the changes, the form, the form container has zero bottom margin the height of the app is exactly the same as the mock, in my computer box-shadow at right-bottom of the main app doesn't appear, you can check out the image in the readme file.
Thanks for the attention.

**Review by afanimartin — APPROVED** [review]
Hi @Rocio01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

### Martin Afani
#### Slack: whiz25

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**